cms/web/practice/js/task.js

```diff
@@ -220,4 +217,18 @@ angular.module('pws.task', [])
     $scope.showDetails = function(id) {
       subsDatabase.subDetails(id);
     };
+  })
+  .directive('pdf', function() {
+    return {
+      restrict: 'E',
+      link: function(scope, element, attrs) {
+        var hasBuiltInPdf = !("ActiveXObject" in window) && !/iPhone|iPod|Android|BlackBerry|Opera Mini|Phone|Mobile/i.test(navigator.userAgent);
+        if (hasBuiltInPdf)
+          element.replaceWith('<object data="' + attrs.src + '" type="application/pdf" class="' + attrs.class + 
+            '">Update your browser!</object>');
+        else
+          element.replaceWith('<iframe seamless src="assets/pdfjs/web/viewer.html?file=' + attrs.src +
+            '" class="' + attrs.class +'"/>');
+      }
+    };
```

Comment:
Ho dovuto creare una directive apposta perché usando `object` direttamente dall'html con chrome non funziona (con ff e edge invece sì). Praticamente chrome non lascia il tempo ad angular di sostituire il campo data con il valore necessario e cerca di accedere a un file che si chiama `{{getPDFURL(task.statements.it)}}` che ovviamente non esiste. Anche dopo che il valore viene aggiornato comunque chrome non aggiorna la schermata, cioè non si aggiorna se modificato il paramentro data del tag, immagino sia un qualche bug di webkit/chromium. Ho anche provato infatti ad usare ng-attr-data, per dare un valore a data solo dopo averlo parsato, ma senza successo.
Usando la directive pdf invece gli si presenta subito l'html con l'attributo valido e non ha quindi problemi nel visualizzarlo.